includes/wc-admin-update-functions.php

```diff
@@ -316,3 +316,41 @@ function wc_admin_update_290_update_apperance_task_option() {
 function wc_admin_update_290_db_version() {
 	Installer::update_db_version( '2.9.0' );
 }
+
+/**
+ * Use woocommerce_admin_activity_panel_inbox_last_read from the user meta to set wc_admin_notes.is_read col.
+ */
+function wc_admin_update_300_update_is_read_from_last_read() {
+	global $wpdb;
+	$meta_key  = 'woocommerce_admin_activity_panel_inbox_last_read';
+	$last_read = $wpdb->get_results(
+		$wpdb->prepare(
+			"
+		select meta_value from {$wpdb->usermeta} 
+		where meta_key=%s
+		order by meta_value desc
+	",
+			$meta_key
+		)
+	);
+
+	if ( count( $last_read ) ) {
+		$date_in_utc = gmdate( 'Y-m-d H:i:s', $last_read[0]->meta_value );
```

Comment:
This results in an error for me.  I think the stored time is coming from JS so it's in milliseconds.  We'll need to update this to:
```suggestion
		$date_in_utc = gmdate( 'Y-m-d H:i:s', intval( $last_read[0]->meta_value ) / 1000 );
```